src/backend/src/routes/users.routes.ts

```diff
@@ -12,7 +13,12 @@ const userRouter = express.Router();
 userRouter.get('/', getAllUsers);
 userRouter.get('/:userId', getSingleUser);
 userRouter.get('/:userId/settings', getUserSettings);
-userRouter.post('/:userId/settings', updateUserSettings);
+userRouter.post(
+  '/:userId/settings',
+  body('defaultTheme').isBoolean().not().isEmpty(),
+  body('slackId').isString().not().isEmpty().optional({ nullable: true }),
```

Comment:
I think `notEmpty()` does the same as `not().isEmpty()` here.

Reply:
i think notEmpty doesn't exist anymore? according to a quick google search idk